src/util/hass-util.html

```diff
@@ -69,6 +71,16 @@
   }
 };
 
+// If root has lastChild, set 'isVisible' attribute of that child to false.
+window.hassUtil.resetDynamicContent = function (root) {
```

Comment:
I tried it with chrome desktop and mobile.

Probably the object is not destroyed when removed. Maybe there is a reference to it somewhere in our code or in Polymer.